ChargerMail/src/System.java

```diff
@@ -0,0 +1,53 @@
+import java.util.ArrayList;
+/**
+ * 
+ * @author tevonwalker
+ * A class for implementing and managing ChargerMail, a simple mail client
+ */
+public class System {
+
+	/**
+	 * Default constructor creates empty system
+	 */
+	
+	public System(){
+		
+	}
+	
+	/**
+	 * Checks to see if userName is already taken
+	 * @param userName Username to test 
+	 * @return a boolean value is returned depending on whether this username is unique or not
+	 */
+	
+	private boolean isNewUserUnique(String userName){
```

Comment:
That's a good point. I have net received any compile errors on my end, but I will change it just in case. I think because we use the 'default' package and not the 'java' package, we have not received compile errors.